pgo/cmd/auth.go

```diff
@@ -175,15 +178,31 @@ func GetCredentialsFromEnvironment() msgs.BasicAuthCredentials {
 	}
 	return creds
 }
-func GetCredentials() {
-	log.Debug("GetCredentials called")
+
+// GetSessionCredentials gathers the pgouser and password information
+// and stores them for use by the PGO client
+func GetSessionCredentials(){
```

Comment:
This is technically not a `Get` this is a `Set` -- I'd recommend we call it `SetSessionUserCredentials` :smile:

Reply:
Makes sense to me, updated!